skbio/stats/power.py

```diff
@@ -821,23 +883,30 @@ def _calculate_power(p_values, alpha=0.05):
 
     """
 
-    w = (p_values < float(alpha)).sum()/float(p_values.shape[0])
+    if p_values.ndim == 1:
+        p_values = p_values * np.array([[1]])
+
+    w = (p_values < float(alpha)).sum(1)/float(p_values.shape[1])
```

Comment:
Looks like axis 1 is being summed here correct?

May make it more clear if `sum(axis=1)` was used instead.